__tests__/app.test.js

```diff
@@ -192,35 +192,119 @@ describe("App", () => {
     test("400: incorrect data type for the review_id param", () => {
       return request(app)
         .post("/api/reviews/abc/comments")
-        .expect(400)
         .send({ username: "mallionaire", body: "This is a test!" })
+        .expect(400)
         .then(({ body: { message } }) => {
           expect(message).toBe("Bad Request");
         });
     });
     test("400: incorrect data type for the username value", () => {
       return request(app)
         .post("/api/reviews/1/comments")
-        .expect(400)
         .send({ username: 100, body: "This is a test!" })
+        .expect(400)
         .then(({ body: { message } }) => {
           expect(message).toBe("Bad Request");
         });
     });
     test("400: request object does not have a username property", () => {
       return request(app)
         .post("/api/reviews/1/comments")
-        .expect(400)
         .send({ body: "This is a test!" })
+        .expect(400)
         .then(({ body: { message } }) => {
           expect(message).toBe("Bad Request");
         });
     });
     test("400: request object does not have a body property", () => {
       return request(app)
         .post("/api/reviews/1/comments")
-        .expect(400)
         .send({ username: "mallionaire" })
+        .expect(400)
+        .then(({ body: { message } }) => {
+          expect(message).toBe("Bad Request");
+        });
+    });
+  });
+
+  describe("PATCH /api/reviews/:review_id", () => {
+    test("200: Responds with an updated review object with the votes incremented correctly", () => {
+      return request(app)
+        .patch("/api/reviews/1")
+        .send({ inc_value: 1 })
```

Comment:
I dont think this is a problem so you dont need to change it but I'm pretty sure the ticket asks for a key of inc_votes not inc_value, should be fine you'll just need to remember when you use your api on FE 👍